webpack.config.js

```diff
@@ -63,18 +64,47 @@ module.exports = {
         ],
       },
       {
-        test: /\.svg$/,
-        loader: 'file-loader',
-        query: {
-          name: 'static/media/[name].[hash:8].[ext]',
-        },
-      },
-      {
-        test: /\.(woff|woff2|eot)$/,
+        test: /\.(woff|woff2|eot|png|jpe?g|gif)$/,
         use: {
           loader: 'url-loader',
         },
       },
+      {
+        test: /\.svg$/,
+        include: [/src\/svgs\//],
+        use: [
+          {
+            loader: 'svg-sprite-loader',
+          },
+          {
+            loader: 'svgo-loader',
+            options: {
+              plugins: [
+                { removeTitle: true },
+                { removeNonInheritableGroupAttrs: true },
+                { removeUselessStrokeAndFill: false },
+                { collapseGroups: true },
+                { convertColors: { shorthex: false } },
+                { convertPathData: false },
+                {
+                  removeAttrs: {
+                    attrs: '(fill|stroke)',
+                    preserveCurrentColor: true,
```

Comment:
this options allows you do something like `<path fill="currentColor"...` and svgo will preserve or not remove those kinds of attributes. Otherwise, the whole point of SVGO is to remove as many attributes uneeded as possible to optimize